network/model.go

```diff
@@ -11,3 +11,8 @@ type CreateInteractionRequest struct {
 type CreateGameRequest struct {
 	PlayerIDs []model.PlayerID `json:"playerIDs"`
 }
+
+type CreatePlayerRequest struct {
+	ID   model.PlayerID `json:"id"`
+	Name string         `json:"name"`
+}
```

Comment:
We'll need to update the test here to match this json. Actually, why don't we just use this network model in that test?

https://github.com/joshprzybyszewski/cribbage/blob/a4d4da6f7dece5daf98ffbf1c8837cf46facf4a9/server/server_test.go#L79-L84